fhir-server-test/src/main/java/org/linuxforhealth/fhir/server/test/TaggingInterceptor.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * (C) Copyright IBM Corp. 2021, 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.linuxforhealth.fhir.server.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.linuxforhealth.fhir.model.resource.Bundle;
+import org.linuxforhealth.fhir.model.resource.Resource;
+import org.linuxforhealth.fhir.model.type.Code;
+import org.linuxforhealth.fhir.model.type.Coding;
+import org.linuxforhealth.fhir.model.type.Meta;
+import org.linuxforhealth.fhir.model.type.Uri;
+import org.linuxforhealth.fhir.persistence.context.FHIRPersistenceEvent;
+import org.linuxforhealth.fhir.server.spi.interceptor.FHIRPersistenceInterceptor;
+import org.linuxforhealth.fhir.server.spi.interceptor.FHIRPersistenceInterceptorException;
+
+/**
+ * A sample persistence interceptor that adds a tag to each resource before it gets persisted.
```

Comment:
Old description? Also adds a tag after reading the resource.

Reply:
```suggestion
 * A sample persistence interceptor that adds a "before" tag to each resource before it gets persisted and an "after" tag when it is read.
```